site/styles/_footer.scss

```diff
@@ -49,6 +49,34 @@ footer {
   }
 }
 
+.social-media-icons {
+  padding-top: 10px;
+  text-align: center;
+  @include media-breakpoint-up(xl) {
+    text-align: left;
+    :first-child {
+      padding-left: 0px;
+      margin-left: -1px;
+    }
+  }
+  
+  .social-media-icon {
```

Comment:
I would also stick the word "footer" in here somehow and extract it from this block (the linter is failing because part of this is going too many layers deep).

Reply:
FYI... you can run the linter with `npm run css:lint`... it will tell you what's up.

Reply:
Fixed